Library/Homebrew/utils/bottles.rb

```diff
@@ -109,7 +109,7 @@ def initialize
 
       def fetch_checksum_for(tag)
         tag = find_matching_tag(tag)
-        return self[tag], tag if tag
+        self[tag]
```

Comment:
What if this doesn't find a matching tag?

Reply:
Fixed now, `if tag` is back.